experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -104,6 +104,18 @@ class Flowables {
     return Flowable<T>::create(std::move(lambda));
   }
 
+  template <typename T>
+  static Reference<Flowable<T>> repeat(T value) {
+    auto lambda = [v = std::move(value)](
+        Subscriber<T> & subscriber, int64_t requested) mutable {
```

Comment:
The rsocket spec forbids request(n) where n <= 0.  That said, we still use `int64_t` because it's a signed int in the spec for compat with Java.

I don't think we should check for `requested <= 0` here because that would suggest a bug in `Flowable::create()`.